### PR TITLE
Unit tests use example input

### LINKED ISSUE #37: Unit test should use example
The unit tests should use the same mapper, reducer, input, and output as the example in the README.  This should match the Hadoop Streaming tutorial.

Right now, they are slightly different, which is confusing.

### PR DESCRIPTION
**Pull request by jaredzh**
Closes #37

### CODE REVIEW

**Comment by jaredzh** [comment]
I updated `map.sh` so that it's case-sensitive in order to be consistent with `map.py`. It causes errors in the tests if `map.sh` is case-insensitive because we are using group by hash which causes the outputs to be different. @awdeorio

File: tests/testdata/word_count/map.sh [hunk 1]
```diff
@@ -9,4 +9,4 @@
 set -Eeuo pipefail
 
 # Map
-cat | tr '[ \t]' '\n' | tr '[:upper:]' '[:lower:]' | awk '{print $1"\t1"}'
+cat | tr '[ \t]' '\n' | awk '{print $1"\t1"}'
```
Comment (awdeorio) [inline tests/testdata/word_count/map.sh:12]: Good catch

File: madoop/__main__.py [hunk 2]
```diff
@@ -87,8 +87,7 @@ def __call__(self, parser, *args, **kwargs):
         src = madoop_dir/"example"
         dst = pathlib.Path("example")
         if dst.exists():
-            print(f"Error: directory already exists: {dst}")
-            parser.exit(1)
+            parser.error(f"directory already exists: {dst}")
```
Comment (awdeorio) [inline madoop/__main__.py:90]: This is just a nitpick I found.

File: tests/test_cli.py [hunk 3]
```diff
@@ -91,12 +91,22 @@ def test_hadoop_arguments(tmpdir):
 def test_example(tmpdir):
     """Example option should copy files."""
     with tmpdir.as_cwd():
-        subprocess.run(["madoop", "--example"], check=True)
+        subprocess.run(
```
Comment (awdeorio) [inline tests/test_cli.py:94]: Another nitpick: suppress the output from the executable during this test.

**Review by awdeorio — APPROVED** [review]
LGTM after pushing a few nitpicks

**Pull request merged**